packages/dds/tree/src/test/simple-tree/tree.spec.ts

```diff
@@ -58,6 +59,45 @@ describe("class-tree tree", () => {
 		view.initialize({ stuff: ["a", "b"] });
 	});
 
+	it("root access leak", () => {
```

Comment:
nit: I think naming the test case after the expectation here is slightly clearer.
```suggestion
	it("accessing view.root does not leak memory", () => {
```

and 'Tree.parent does not leak memory' or something like that below.

Reply:
I thought there was a leak when I wrote that so I had the wrong expectation. Good catch.